sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -221,6 +221,8 @@ case class AdaptiveSparkPlanExec(
       .map(_.toLong).filter(SQLExecution.getQueryExecution(_) eq context.qe)
   }
 
+  private[sql] lazy val finalPlan: SparkPlan = getFinalPhysicalPlan()
```

Comment:
can we just change access of `getFinalPhysicalPlan()` to `private[sql]` instead? I don't think we need a duplicate of `executedPlan` in this class.

Reply:
+1

Reply:
Will do.